go/libraries/doltcore/branch_control/branch_control.go

```diff
@@ -56,22 +66,86 @@ func init() {
 
 // CreateController returns an empty *Controller.
 func CreateController(ctx context.Context) *Controller {
-	accessTbl := newAccess("", "")
-	return &Controller{
-		Access:    accessTbl,
-		Namespace: newNamespace(accessTbl, "", ""),
-	}
+	return CreateControllerWithSuperUser(ctx, "", "")
 }
 
 // CreateControllerWithSuperUser returns a controller with the given user and host set as an immutable super user.
 func CreateControllerWithSuperUser(ctx context.Context, superUser string, superHost string) *Controller {
+	//TODO: put in the context
 	accessTbl := newAccess(superUser, superHost)
 	return &Controller{
 		Access:    accessTbl,
 		Namespace: newNamespace(accessTbl, superUser, superHost),
 	}
 }
 
+// LoadData loads the data from the given location into the controller.
+func LoadData(ctx context.Context, branchControlFilePath string, doltConfigDirPath string) error {
+	//TODO: load into the context's controller
+	if !enabled {
+		return nil
+	}
+
+	// Do not attempt to load from an empty file path
+	if len(branchControlFilePath) == 0 {
+		return nil
+	}
+
+	StaticController.branchControlFilePath = branchControlFilePath
```

Comment:
Eventually we probably want a way to bake this down to a single file that has references to multiple DBs and their permissions, so that it's possible to e.g. push it to another host during replication